pkg/ccl/streamingccl/streamingest/stream_ingestion_dist.go

```diff
@@ -150,8 +168,35 @@ func startDistIngestion(
 		func() time.Duration { return replanFrequency.Get(execCtx.ExecCfg().SV()) },
 	)
 
+	// TODO(adityamaru): Replace this timer-based loop once we teach requesting
```

Comment:
what do you mean by "timer-based loop"? This looks like a timer based loop to me?

Reply:
Yeah, the TODO is to _replace_ the time-based loop with a fetch model once the requesting of execution details once each resumer can be contacted to be provide its relevant details.

Reply:
lol, I totally missed the "replace" in that docstring.